File: 0168-excel-sheet-column-title/0168-excel-sheet-column-title.java
class Solution {

    public String convertToTitle(int n ) {
        StringBuilder ans = new StringBuilder();
        while (n > 0) {
            --n;
            int d = n % 26;
            n /= 26;
            ans.append((char) ('A' + d));
        }
        ans.reverse();
        return ans.toString();
    }
}

File: 0168-excel-sheet-column-title/0168-excel-sheet-column-title.cpp
class Solution
{
    public:
        string convertToTitle(int n)
        {
            string ans = "";
            while (n > 0)
            {
                --n;
                int d = n % 26;
                n /= 26;
                char c = 'A' + d;
                ans =c+ ans;
            }
            return ans;
        }
};